lib/addons/cloud-watch-insights/index.ts

```diff
@@ -1,14 +1,13 @@
-import {Construct} from 'constructs';
-import * as iam from "aws-cdk-lib/aws-iam";
+import {Construct, IConstruct} from 'constructs';
+import {IManagedPolicy, ManagedPolicy} from "aws-cdk-lib/aws-iam";
 import {ClusterInfo, Values} from "../../spi";
-import {conflictsWith, supportsALL} from "../../utils";
+import {conflictsWith, createNamespace as makeNamespace, supportsALL} from "../../utils";
 import {CoreAddOn, CoreAddOnProps} from "../core-addon";
 import {ebsCollectorPolicy} from "./iam-policy";
-import {IManagedPolicy, ManagedPolicy} from "aws-cdk-lib/aws-iam";
-import {KubernetesVersion, ServiceAccount} from "aws-cdk-lib/aws-eks";
+import {KubernetesVersion} from "aws-cdk-lib/aws-eks";
 
 const versionMap: Map<KubernetesVersion, string> = new Map([
-  [KubernetesVersion.V1_29, "v1.3.1-eksbuild.1"],
+  [KubernetesVersion.V1_29, "v1.4.0-eksbuild.1"],
```

Comment:
Default version for `1.29` is `v1.5.5-eksbuild.1`. Likewise please check for other 4 versions.

Reply:
+1